tsdb/isolation.go

```diff
@@ -131,6 +148,10 @@ func (i *isolation) State(mint, maxt int64) *isolationState {
 // function on those states. The given function MUST NOT mutate the isolationState.
 // The iteration is stopped when the function returns false or once all reads have been iterated.
 func (i *isolation) TraverseOpenReads(f func(s *isolationState) bool) {
+	if i.disabled {
+		return
+	}
```

Comment:
We _need_ to traverse on open reads :) disabling isolation would not disable tracking of reads